cmd/strelaypoolsrv/gui/index.html

```diff
@@ -185,9 +191,8 @@ <h1>Relay Pool Data</h1>
 
 
     <script type="text/javascript" src="//code.jquery.com/jquery-2.1.4.min.js"></script>
-    <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/angular.js/1.5.8/angular.min.js"></script>
+    <script type="text/javascript" src="http://cdnjs.cloudflare.com/ajax/libs/angular.js/1.5.8/angular.min.js"></script>
```

Comment:
no `http`, either `//...` as it was to keep whatever the browser is using, or `https` if we need to hardcode something for some reason